### ISSUE TITLE
startLines in tag-lines is not a replacement of newline-after-description when there is no description

### ISSUE THREAD

**Issue by stof** [gajus/eslint-plugin-jsdoc#1022]
## Expected behavior

If there is no description, `tag-lines` should have a way to allow the tag to be at the beginning of the jsdoc, even though we want a line between the description and the tags when we have both. Otherwise, there is no proper replacement for `jsdoc/newline-after-description` with `always`

## Actual behavior

`startLines: 1` always forces an empty line even if it is the beginning of the jsdoc.

## ESLint Config

```js
{
    "plugins": ["jsdoc"],
    "rules": {
        "jsdoc/tag-lines": ["error", "never", {
            "startLines": 1
        }],
    }
}
```

## ESLint sample

```js

/**
 * @param {string} input
 * @param {string|rename.Options} renamingConfig
 * @returns {gulp.TaskFunction}
 */
function processSass (input, renamingConfig) {
    // <redacted>
}
```

## Environment

- Node version: 18.16.0
- ESLint version: 8.38.0
- `eslint-plugin-jsdoc` version: 43.0.0

**Comment by brettz9** [comment]
Sorry about that, and thanks for the report. `startLines` was indeed intended to only apply after any text (as with `newline-after-description`), so #1023 will hopefully address that.
